examples/twittermap/web/public/javascripts/map/controllers.js

```diff
@@ -609,8 +615,11 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common','clou
         }
         if ($scope.status.logicLevel === 'city') {
             loadCityJsonByBound(onEachFeature, false);
+        } else {
+          resetGeoIds($scope.bounds, geoData, $scope.status.logicLevel + "ID");
```

Comment:
resetGeoIds are required for both state and county level, too. I think. Check the original line 613.

Reply:
I think this "else" is actually for both "state" and "county" level.